addons_rksv/pos_rksv/static/src/js/screens.js

```diff
@@ -164,7 +166,8 @@ odoo.define('pos_rksv.screens', function (require) {
      */
     screens.ReceiptScreenWidget.include({
         handle_auto_print: function() {
-           console.log('I WANT TO PRINT LATER')
+            if (this.pos.config.iface_rksv)
```

Comment:
Well, it is about delaying the print which is crucial for auto print on local printer setups otherwise you will miss the displayed QR Image before sent to the printer (depending on the speed of rendering locally)